### ISSUE TITLE
[Dev asks] Station modules blueprint and feedback!

### ISSUE THREAD

**Issue by tsunamayo** [tsunamayo/Starship-EVO#2303]
Copy pasted from the other post here is what I am looking to:

a) some basic connecting elements: mini hug with connector on all side, corridors.
b) Shipyard! Some stuff like the old one, some other concept like you want. Please make it so that the ship area is cubic. Interior should have mechanical / servicing stuff (not living quarter). I will add an elevator working like the connector for entering inside the shipyard!
c) A hub for gameplay. Some stuff where I will add mission board and other gameplay elements.
d) Some random living quarter. For immersion!

Also if you have any feedback! Does the scale and the outside feels weird? I am still having doubt, it is okay for modules to be spawned by the game, but for player made content it might be too complex...
Thanks

**Comment by Aiwii** [comment]
Can I paint the modules however or do you want a certain colour scheme?

**Comment by Womble-TC** [comment]
Here's a WIP little tube: 
![image](https://user-images.githubusercontent.com/29961381/83463095-e68e6d00-a464-11ea-9822-f6abf8023830.png)
The interior is for some reason offset by a little bit after I saved + reloaded though:
![image](https://user-images.githubusercontent.com/29961381/83463163-1d648300-a465-11ea-830d-f8ebf70ee202.png)
[Tube_1.zip](https://github.com/tsunamayo/Starship-EVO/files/4713856/Tube_1.zip)

**Comment by Aiwii** [comment]
A bunch of work in progress modules (no interiors)
[Varietypack.zip](https://github.com/tsunamayo/Starship-EVO/files/4713899/Varietypack.zip)

Small Pad
![image](https://user-images.githubusercontent.com/53021785/83464316-c1323c80-a42d-11ea-988c-66feae9438b6.png)

Large Pad
![image](https://user-images.githubusercontent.com/53021785/83464647-c2179e00-a42e-11ea-9dae-be3124db5075.png)

Random module (I'm thinking of doing a restaurant)
![image](https://user-images.githubusercontent.com/53021785/83464346-dc04b100-a42d-11ea-9cb7-50722448cf4c.png)

2 types of half rings (need a lot of work might be scrapped and redone)
![image](https://user-images.githubusercontent.com/53021785/83464392-fe96ca00-a42d-11ea-8903-96d29fb1ce31.png)

2 short tubes / hallways
![image](https://user-images.githubusercontent.com/53021785/83464448-21c17980-a42e-11ea-82a3-4add96eafd61.png)

A large four-way Junction
![image](https://user-images.githubusercontent.com/53021785/83464490-4a497380-a42e-11ea-94bf-6d5f513c2dc8.png)

A small four-way Junction
![image](https://user-images.githubusercontent.com/53021785/83464535-5f260700-a42e-11ea-8716-d105e6ac0579.png)

A large hallway built for the rings but works well for on its own
![image](https://user-images.githubusercontent.com/53021785/83464579-7bc23f00-a42e-11ea-95cc-9e77cdaa778c.png)

an example station I built
![image](https://user-images.githubusercontent.com/53021785/83464600-95fc1d00-a42e-11ea-8aa4-9aa0c2d48b0c.png)

**Comment by Dwarf-LordPangolin** [comment]
**Feedback**
The scale and the feel of the difference between the inside & outside are actually great; I can make a nice big exterior and a  It's definitely not too complex; Space Engineers players have actually been asking Keen Software House to implement something similar to this for years, so being able to do this is basically a dream come true.

**Modules**
I'll just keep updating this as I make new blueprints and update old ones. :)

Corridor, 2-way
[Station Module - Hall 2.zip](https://github.com/tsunamayo/Starship-EVO/files/4714052/Station.Module.-.Hall.2.zip)
![20200601192411_1](https://user-images.githubusercontent.com/33041786/83466695-81735100-a43d-11ea-84ff-12d833a88a12.jpg)

Corridor, 3-way
[Station Module - Hall T.zip](https://github.com/tsunamayo/Starship-EVO/files/4714054/Station.Module.-.Hall.T.zip)
![20200601192415_1](https://user-images.githubusercontent.com/33041786/83466698-833d1480-a43d-11ea-8fc1-0d1ede5b038e.jpg)

Corridor, 4-way
[Station Module - Hall 4.zip](https://github.com/tsunamayo/Starship-EVO/files/4714056/Station.Module.-.Hall.4.zip)
![20200601192418_1](https://user-images.githubusercontent.com/33041786/83466700-8506d800-a43d-11ea-90c4-02af8a40cc0b.jpg)

**Comment by tsunamayo** [comment]
@Dwarf-LordPangolin you can do a 6 way too, I want to add some verticality to it! They could be in 0g.
Looking good all!

**Comment by Womble-TC** [comment]
@tsunamayo any word on the connector model tweaks? There's no rush, but it would nice to have something to plan around.

**Comment by tsunamayo** [comment]
@Womble-TC yes I want to do it!

**Comment by Womble-TC** [comment]
epic

**Comment by Dwarf-LordPangolin** [comment]
Yep, a 6-way is on my list! These are just the ones I've gotten done so far. Thanks for letting us know that we can handle that with 0g; I was trying to figure out how to do it with ladders.

**Comment by Dwarf-LordPangolin** [comment]
The bug mentioned [here](https://github.com/tsunamayo/Starship-EVO/issues/2302) has corrupted my 3-way, 4-way, and 6-way modules. I will be rebuilding them from scratch, so that's going to take a wee bit longer than I'd hoped; going to see if I can find a workaround for the bug.

**Comment by Hutzlipuh** [comment]
[Station-Blueprints-Hutzlipuh.zip](https://github.com/tsunamayo/Starship-EVO/files/4747903/Station-Blueprints-Hutzlipuh.zip)
Here are my Station modules for you , tsunamayo...
hope you can use them .
![20200608202902_1](https://user-images.githubusercontent.com/33233466/84067684-10e1ac80-a9c8-11ea-9c2d-e90f66f2cd19.jpg)
